Machine_Learning/project.py

```
@@ -0,0 +1,115 @@
+        """
+        response = requests.get("http://overpass-api.de/api/interpreter", params={"data": query})
+        hospitals = response.json()['elements']
+        results = []
+        for h in hospitals:
+            name = h.get("tags", {}).get("name", "Unnamed Hospital")
+            hlat = h.get("lat") or h.get("center", {}).get("lat")
+            hlon = h.get("lon") or h.get("center", {}).get("lon")
+            if hlat and hlon:
+                results.append((name, hlat, hlon))
+        return results
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)